packages/wrangler/src/__tests__/standard-pricing.test.ts

```diff
@@ -37,6 +37,16 @@ describe("standard-pricing", () => {
 	runInTempDir();
 	const std = mockConsoleMethods();
 
+	// TODO: remove the fake timers and irrelevant tests after March 1st
```

Comment:
Please can you create a WIP PR for removing these so that we can simply land that PR when we hit March? Otherwise this comment is likely to be lost in the mists of time!

Reply:
WIP PR: https://github.com/cloudflare/workers-sdk/pull/5035